app/src/main/cpp/skyline/audio.h

```diff
@@ -0,0 +1,145 @@
+#pragma once
+
+#include <queue>
+#include <oboe/Oboe.h>
+
+#include <kernel/types/KEvent.h>
+#include "common.h"
+
+namespace skyline::audio {
+    enum class PcmFormat {
+        Invalid = 0,
+        Int8 = 1,
+        Int16 = 2,
+        Int24 = 3,
+        Int32 = 4,
+        PcmFloat = 5,
+        AdPcm = 6
+    };
+
+    enum class AudioOutState {
+        Stopped = 0,
+        Started = 1
+    };
```

Comment:
enum classes like this should specify a underlying type such as `enum class PcmFormat : u32`.